corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,18 +34,22 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
 {% endblock %}
 
 {% block page_content %}
-    <table id="data-dictionary-table" class="table">
-        <thead>
-            <th>Case Property</th>
-            <th>Type</th>
-            <th>Description</th>
-        </thead>
-        <tbody data-bind="foreach: activeCaseTypeData">
-            <tr>
-                <td data-bind="text: name"></td>
-                <td data-bind="text: type"></td>
-                <td data-bind="text: description"></td>
-            </tr>
-        </tbody>
-    </table>
+    <div class="row">
+        <div class="col-md-10">
+            <table id="data-dictionary-table" class="table">
+                <thead>
+                    <th>{% trans "Case Property" %}</th>
+                    <th>{% trans "Type" %}</th>
```

Comment:
This table isn't sortable right now, and the UX is not complete. I'm just including this as a quick way to view it while I was developing the backend. 

It is "data type" https://github.com/dimagi/commcare-hq/pull/13957/files#diff-ff11fdee49a311b27e26488e60a55f85R3 so case property should be on the left as it's the "unique key" and type is more information about the property. This table only has one case type per page